ogre_mesh_viewer.py

```diff
@@ -339,7 +396,7 @@ def preRenderTargetUpdate(self, evt):
                     changed = False
                     if astate.getLength() > 0:
                         ImGui.SameLine()
-                        changed, value = ImGui.SliderFloat("", astate.getTimePosition(), 0, astate.getLength(), "%.3fs")
+                        changed, value = ImGui.SliderFloat("", astate.getTimePosition(), 0, astate.getLength(), "%.2fs")
```

Comment:
unrelated change? Please keep .3f

Reply:
True, but is it necessary to have so much precision.?

It looks bad and reformats the text too much when playing.